LBH.AdultSocialCare.Api.Tests/Extensions/ICollectionExtensions.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+
+namespace LBH.AdultSocialCare.Api.Tests.Extensions
+{
+    // ReSharper disable once InconsistentNaming
+    public static class ICollectionExtensions
+    {
+        public static ICollection<T> AddRange<T>(this ICollection<T> collection, IEnumerable<T> items)
+        {
+            foreach (var item in items)
+            {
+                collection.Add(item);
+            }
+
+            return collection;
+        }
```

Comment:
Hmm. Almost asked why but quickly found out the reason

Reply:
mostly to chain multiple AddRange calls